src/ring/middleware/session_timeout.clj

```diff
@@ -4,6 +4,27 @@
 (defn- current-time []
   (quot (System/currentTimeMillis) 1000))
 
+(defn- session-idle-expired?
+  [request]
```

Comment:
Where there's space, can you put the arguments on the same line as `defn`? If there's no need to use up extra whitespace, we can write:

```clojure
(defn- function [arg]
  ...)
```

Instead of:

```clojure
(defn- function
  [arg]
  ...)
```